fapolicy_analyzer/ui/profiler_page.py

```diff
@@ -73,9 +85,26 @@ def __init__(self, fapd_manager):
     def analyze_button_sensitivity(self):
         return self.analysis_available
 
+    def on_next_system(self, system):
+        profilerDict = system["profiler"]
+        self.update_field_text(profilerDict)
```

Comment:
I'd suggest you keep `profilerDict` at the class level and then only call `update_field_text` if that dict is different than what you getting back from the `system` object here. It doesn't seem to be causing any issues, but the code the way you have it will do an extra call to `update_field_text` when you `dispatch(set_profiler_state(entryDict))` below.